RecoTauTag/RecoTau/plugins/RecoTauCleaner.cc

```diff
@@ -336,7 +330,11 @@ RecoTauCleanerImpl<reco::PFTauCollection>::fillDescriptions(edm::ConfigurationDe
       cleaner_Charge.addParameter<std::string>("plugin", "RecoTauChargeCleanerPlugin");
       cleaner_Charge.addParameter<int>("passForCharge", 1);
       cleaner_Charge.addParameter<double>("selectionFailValue", 0);
-      cleaner_Charge.addParameter<std::vector<unsigned int>>("nprongs", {1, 3,});
+      cleaner_Charge.addParameter<std::vector<unsigned int>>("nprongs",
+                                                             {
+                                                                 1,
+                                                                 3,
+                                                             });
```

Comment:
this doesn't seem like a justified split; as if the trailing comma took it over the 80 char limit